src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1746,30 +1807,37 @@ public function getLanguages()
             return $this->languages;
         }
 
-        $languages = AcceptHeader::fromString($this->headers->get('Accept-Language'))->all();
+        $languageHeaderValue = $this->headers->get('Accept-Language');
+        $languages = array_keys(AcceptHeader::fromString($languageHeaderValue)->all());
+
+        $delimiter = '-';
         $this->languages = [];
-        foreach ($languages as $lang => $acceptHeaderItem) {
```

Comment:
Please revert the change in this file: the code works fine, changing them for CS reasons can lead to introducing bugs. We already merged similar changes and broke things by mistake.

Reply:
no, I won't revert. 

if you have the code, covered with tests, that you are 'afraid to touch, because something can go wrong' (no matter the reason of changes, whether it is code style or functionality) - you have either a bad code or lack of tests. 

if you looked at that code part, you would notice the variable $acceptHeaderItem was never used. but no, let's leave it as is. good approach! but not good enough for me, sadly.